### PR TITLE
Source Zendesk Support: Add stream: `Post Comments`

### LINKED ISSUE #16882: Source Zendesk Support: new stream Post Comments
## Tell us about the problem you're trying to solve
[Post Comments](https://developer.zendesk.com/api-reference/help_center/help-center-api/post_comments/). Users can participate in community discussions by adding comments to [community posts](https://developer.zendesk.com/api-reference/help_center/help-center-api/posts/) in the help center. The user must have a Zendesk account and their requests must be authenticated. Anonymous users can't add comments.

[Allowed for:](https://developer.zendesk.com/api-reference/help_center/help-center-api/post_comments/#allowed-forr)
- Agents
- End users

## Describe the solution you’d like
Research pros and cons for implementing this stream.
_Research should contain: what stream should be implemented, permissions and scopes (if needed), paid plan (if needed)._

### PR DESCRIPTION
**Pull request by artem1205**
## What

Resolve https://github.com/airbytehq/airbyte/issues/16882

## How

 Add new stream: `Post Comments`

## Recommended reading order
1. `stream.py`

## 🚨 User Impact 🚨

no breaking changes.

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`1ea93a8bf1`](https://github.com/airbytehq/airbyte/pull/27961/commits/1ea93a8bf109ee061efe057fa7c3a886b1f7d6cd)) - ❌

⏲️  Total pipeline duration: 58.10s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ❌        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688538984/1ea93a8bf109ee061efe057fa7c3a886b1f7d6cd/source-zendesk-support/0.8.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`1ea93a8bf1`](https://github.com/airbytehq/airbyte/pull/27961/commits/1ea93a8bf109ee061efe057fa7c3a886b1f7d6cd)) - ❌

⏲️  Total pipeline duration: 54.81s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ❌        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688539120/1ea93a8bf109ee061efe057fa7c3a886b1f7d6cd/source-zendesk-support/0.8.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`b4d58c9a14`](https://github.com/airbytehq/airbyte/pull/27961/commits/b4d58c9a14120c720d36a2b6b31fb88f3dd9152f)) - ✅

⏲️  Total pipeline duration: 16mn49s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688551563/b4d58c9a14120c720d36a2b6b31fb88f3dd9152f/source-zendesk-support/0.9.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ acceptance_tests:
           # This stream is only available for enterprise accounts https://developer.zendesk.com/api-reference/ticketing/account-configuration/audit_logs/
           - name: "audit_logs"
             bypass_reason: "no records"
+          - name: "post_comments"
+            bypass_reason: "can not populate"
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:35]: why we can not populate this stream?

Comment (artem1205) [reply airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:35]: posts (and comments for them) are available in zendesk community only. We have no subscription for this product yet.

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`2b78c77084`](https://github.com/airbytehq/airbyte/pull/27961/commits/2b78c77084b8a4b467b09978d9bbfdd15a54e810)) - ❌

⏲️  Total pipeline duration: 17mn03s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ❌        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688571528/2b78c77084b8a4b467b09978d9bbfdd15a54e810/source-zendesk-support/0.9.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`8adb6bf5ff`](https://github.com/airbytehq/airbyte/pull/27961/commits/8adb6bf5ff1208f0b6296f6d60dad3bf0638ed12)) - ❌

⏲️  Total pipeline duration: 17mn51s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688579213/8adb6bf5ff1208f0b6296f6d60dad3bf0638ed12/source-zendesk-support/0.9.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`13b955ee97`](https://github.com/airbytehq/airbyte/pull/27961/commits/13b955ee978c74b0887b9ff35385829b9722c1d8)) - ✅

⏲️  Total pipeline duration: 17mn24s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_source-zendesk-support-16882/1688585244/13b955ee978c74b0887b9ff35385829b9722c1d8/source-zendesk-support/0.9.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Review by lazebnyi — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**